playground/TestShop/AppHost/Properties/launchSettings.json

```diff
@@ -4,24 +4,35 @@
       "commandName": "Project",
       "launchBrowser": true,
       "dotnetRunMessages": true,
-      "applicationUrl": "https://localhost:15887;http://localhost:15888",
+      "applicationUrl": "http://localhost:15888",
```

Comment:
Updated the TestShop `launchSettings.json` to better match what the templates will do.